data/pages.yaml

```diff
@@ -132,16 +132,18 @@ maintainer:
     documentation: Look after collection documentation
   build_community:
     intro: Community maintainers play a special role to enable and encourage contributors.
-    communication: Join or start a community discussion
+    grow_community: Explore ways how to grow your community
+    documentation: Maintain good contributor documentation
     contributor_paths: Understand Ansible contributor paths
-    encourage: Attract and mentor new contributors
   collection_inclusion:
     intro: Satisfy requirements to get collections included in the Ansible package.
     process: Understand the inclusion process
-    review: Go through inclusion review cycles
+    review: Review other inclusion requests
+    submit: Submit your collection for inclusion
   participate:
     intro: Get involved in higher-level decision processes that guide the broader community.
-    join: Join the Ansible steering committee
+    community_topics: Discuss and vote on Community topics
+    join: Join the Ansible community steering committee
```

Comment:
I made the pages follow the user journeys as closely as possible. I'm not sure about adding items to persona pages that don't map to their journey.

Not saying you're wrong about adding "community topics" but I'd like to make sure that is reflected in the journey because that is what we're basing the pages on: https://github.com/ansible/docsite/blob/personas/user-journeys/ansible-user-journey-maps.md#community-maintainer

Reply:
I believe it's a very important step we missed in https://github.com/ansible/docsite/blob/personas/user-journeys/ansible-user-journey-maps.md#community-maintainer
Do I need to create a PR to add it to `ansible-journey-pams.md`?

- community-topics affect community collections and the package, resp they affect maintainers
- "Get involved in higher-level decision processes that guide the broader community.": community-topics is a place where the decisions are made and anyone is encourage to open topics, discuss and vote (not only SC)
- to be active in the topics is one of not many ways how to get to the committee.

Reply:
Thanks for catching that @Andersson007 Yes, please do sync the changes here with those journeys